internal/config/loader.go

```diff
@@ -116,6 +119,38 @@ func LoadConfig(useRegistry bool, profile string, confDir string) (*common.Confi
 	return configuration, nil
 }
 
+func parseUseRegistry(useRegistry string, config *common.Config) bool {
+	switch useRegistry {
```

Comment:
minor nitpick, this type of operation reads more cleanly as 

```go
switch strings.ToLower(useRegistry) {
    case "false":
        doTheFalseThing()
    case "true":
        doTheTrueThing()
    default:
        doTheDefaultThing()
}
```

Reply:
thanks, that's good idea